package/gluon-mesh-batman-adv-brmldproxy/files/etc/hotplug.d/iface/50-gluon-mesh-batman-adv-brmldproxy

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+([ "$INTERFACE" != "bat0" ] || [ "$ACTION" != "ifup" ]) && exit 0
+
+[ "$(gluon-show-site | jsonfilter -e "@.mesh.filter_membership_reports")" = "false" ] && exit 0
+
+[ -x /etc/init.d/brmldproxy ] && /etc/init.d/brmldproxy enabled && \
```

Comment:
- There is no need to check for the executability, `gluon-mesh-batman-adv-brmldproxy` depends on `brmldproxy`
- There is no need to check for the `enabled` status (on Gluon, all installed services are always enabled; disabling is not preserved on ugprades, so it is not supported). If disabling should be possible, a `/etc/config/gluon` flag must be introduced.